src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -118,11 +87,65 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
     imageDownloadBtn,
   } = lang;
 
+  const getLimits = async (request: DatapointsMultiQuery) => {
+    let { start = 0, end = 0 } = request;
+
+    const project = context!.project || '';
+    const url = `/api/v1/projects/${project}/timeseries/data/latest`;
```

Comment:
why you don't use `retreaveLatest` from sdk instead?
ref: https://cognitedata.github.io/cognite-sdk-js/classes/datapointsapi.html#retrievelatest

Reply:
Will do!